### PR TITLE
Cleanup host file

### PR DESCRIPTION
**Pull request by otherview**

### CODE REVIEW

File: go/obscuronode/host/host.go [hunk 1]
```diff
@@ -17,32 +17,18 @@ import (
...
-	BroadcastRollup(r obscurocommon.EncodedRollup)
-	BroadcastTx(tx nodecommon.EncryptedTx)
-}
-
-// ClientServer is the layer responsible for handling requests from Obscuro client applications.
-type ClientServer interface {
-	Start()
-	Stop()
-}
+const ClientRPCTimeoutSecs = 5
```
Comment (joeldudley) [inline go/obscuronode/host/host.go:20]: @otherview You should use the value from the host's config for this.

File: go/obscuronode/host/host.go [hunk 2]
```diff
@@ -17,32 +17,18 @@ import (
...
 
-type StatsCollector interface {
-	// L2Recalc - called when a node has to discard the speculative work built on top of the winner of the gossip round.
-	L2Recalc(id common.Address)
-	NewBlock(block *types.Block)
-	NewRollup(node common.Address, rollup *nodecommon.Rollup)
-	RollupWithMoreRecentProof()
+// todo - this has to be replaced with a proper cfg framework
+type AggregatorCfg struct {
```
Comment (joeldudley) [inline go/obscuronode/host/host.go:23]: @otherview This was removed

File: go/obscuronode/host/host.go (hunk 1 above)
Comment (otherview) [reply go/obscuronode/host/host.go:20]: Merge issue. Fixed 👍

**Review by joeldudley — APPROVED** [review]
LGTM

**Pull request merged**